index.js

```diff
@@ -227,6 +236,29 @@ function formatChannelToRow (c, i) {
   ]
 }
 
+async function validateAddress (server, address) {
+  const api = new RippleAPI({ server })
+  await api.connect()
+  const accountInfo = await api.getAccountInfo(address).catch((err) => {
+    if (err.message !== 'actNotFound') throw err
+    throw new Error('Address "' + address + '" does not exist on ' + server)
+  })
+  const {validatedLedger: {
+    reserveBaseXRP,
+    reserveIncrementXRP
+  }} = await api.getServerInfo()
+
+  const minBalance = (+reserveBaseXRP) +
+    (+reserveIncrementXRP) * accountInfo.ownerCount +
+    (+reserveIncrementXRP) + // reserve for the channel
+    10 + // OUTGOING_CHANNEL_DEFAULT_AMOUNT_XRP from ilp-plugin-xrp-asym-client
```

Comment:
We should really get this from ilp-plugin-xrp-asym-client, which is a dependency of this module I believe.

I would add an export in ilp-plugin-xrp-asym-client and then import it here.